examples/federated-connections/langchain/src/lib/auth0-ai.ts

```diff
@@ -0,0 +1,8 @@
+import { Auth0AI } from "@auth0/ai-langchain";
+
+const auth0AI = new Auth0AI();
+
+export const withCalendarFreeBusyAccess = auth0AI.withFederatedConnection({
```

Comment:
```suggestion
export const withTokenForGoogleConnection = auth0AI.withFederatedConnection({
```

Reply:
The purpose of this naming is that not only it ensures that there is a token in the context for the googles apis but also that it has the right scopes.